### ISSUE TITLE
Must support `a=rtcp-fb:*` when checking RTP capabilities

### ISSUE THREAD

**Issue by ibc** [versatica/mediasoup-client#239]
`a=rtcp-fb:*` means that all codecs in this `m=` section support the RTCP features exposed in this attribute line. Purpose of it is to save space in the SDP by reducing number of `a=rtcp-fb` lines per `m=` section. libwebrtc will support this soon so we must be ready in **mediasoup-client** and **libmediasoupclient**. 

**Example:**
`a=rtcp-fb:* transport-cc` means that all codecs in this `m=` section support `transport-cc`.

**NOTE:** This is a SDP thing. We transmit ORTC parameters from client to mediasoup and vice-versa. AFAIU we don't need to support this neither in mediasoup side nor in the ORTC parameters, utils and definitions within mediasoup-client and libmediasoupclient.

So basically, when we parse a SDP and generate `RtpParameters` or `RtpCapabilities` must be ready to manage `a=rtcp-fb:*` lines and apply their supported RTCP features into all codecs within the generated `RtpParameters` or `RtpCapabilities`. This is just an estimation, things may need to be different.

**Comment by ibc** [comment]
Related: https://github.com/otalk/sdp/pull/89/files

**Comment by ibc** [comment]
PR done: https://github.com/versatica/mediasoup-client/pull/240
